lua/mappings.lua

```diff
@@ -24,6 +24,45 @@ map("v", "x", [=[ "_x ]=], opt)
  this line too ]]
 --
 
+-- toggle unix readline's keymap
+map("n", user_map.unix_keymap.toggle_unix_keymap,"<cmd>lua require 'mappings'.unix_keymap()<CR>",{nowait = true})
```

Comment:
Hmmm, I think rather than a mapping to enable & disable these mappings, maybe a `chadrc.lua` option 

That way a user can set the option in `chadrc.lua`